en/ch06.md

```diff
@@ -3,10 +3,10 @@
 
 The `Transactions` tab let you walk through the transactions sent and received on your account.
 
-(1) **Search:** Quickly find a transaction using any field to look up from.  
-(2) **Filters:** You can choose to filter out the transaction between specific dates, block height or amount. This can be useful if your wallet has tons of transactions.  
-(3) **Transaction list:** Here are displayed your transactions on this account. Each transaction is displayed with it's informations: `Amount`, `Tx ID`, `Height`, `Date`, `Confrimations`, `Fee` (for outgoing transactions), and `Description` *Optional*.  
-(4) **Details:** Clicking here will show you the transaction details: `Tx ID`, `Address Label`, `Address`, and `Payment ID`.  
+(1) **Search:** Quickly filter out transactions from the list searching by transaction ID, destination address, description, amount or blockheight (number of the block in which the transaction was mined).
+(2) **Sort & Filter:** Here you can filter out transactions between specific dates, or sort them by blockheight, date or amount.  
+(3) **Transaction list:** This area displays the sent and received transactions of the currently selected account. Each transaction is displayed with the following information: `Amount`, `Blockheight`, `Date`, `Fee` (for outgoing transactions), `Confirmations`, `Description` (optional), `Transaction ID`, `Transaction key`, and `Destination address`.  
+(4) **Details:** Clicking here will show you the following transaction details: `Tx ID`, `Address Label`, `Address`, and `Payment ID`.  
 (5) **Prove transaction:** Clicking here will generate a payment proof in case of a dispute. More details on proving payments can be found [here](https://getmonero.org/resources/user-guides/prove-payment.html) *For outgoing transaction only*  
-(6) **Set description:** You can optionally click here to set a description of your choice to this transaction.  
-(7) **Advanced:** Lets you choose in which formate the date should be displayed (you probably want to leave this option as it is).
+(6) **Set description:** Click on this icon to set a new description or edit an existing description for this transaction.  
```

Comment:
```suggestion
(6) **Set description:** Click on this icon to set a new description or edit an existing one for this transaction.  
```
To avoid the repetition of 'description'